src/charm.py

```diff
@@ -165,6 +155,14 @@ def ingress_config(self):
             ingress_config["tls-secret-name"] = tls_secret_name
```

Comment:
Should we set the secret here? This can be directly configured directly the ingress without having wordpress sitting in the middle, providing a more standard way across deployments (and easier to automate in the future)

Reply:
The original version of the charm has a config called `tls_secret_name`, this is solely for the backward compatibility's sake. Maybe I will remove this in the future.